Gruntfile.js

```diff
@@ -493,6 +495,7 @@ module.exports = function (grunt) {
         'copy:e2e',
         'copy:e2e-pages',
         'exec:build-e2e-app',
+        'exec:build-e2e-dev-app',
```

Comment:
What is actually happening here? Is it overwriting the production app bundle with the dev bundle?